app/views/lead_providers/partnership_csv_uploads/errors.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for :title, "CSV results" %>
+<% content_for :before_content, govuk_back_link(text: "Back", href: new_lead_providers_report_schools_partnership_csv_uploads_path) %>
+<h1 class="govuk-heading-xl">Your CSV has errors</h1>
+
+<p class="govuk-body">We have found <%= @urns.count %> rows in your CSV:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <% if @errors.count == 1 %>
+    <li><%= @errors.count %> row has errors</li>
+  <% else %>
+    <li><%= @errors.count %> rows have errors</li>
+  <% end %>
+
+  <% if (@valid_schools.count) == 1 %>
+    <li><%= @valid_schools.count %> row matches available schools</li>
+  <% else %>
+    <li><%= @valid_schools.count %> rows match available schools</li>
+  <% end %>
```

Comment:
Rails likes to pretend it knows how English works, is there a nicer way to pluralise those?

Reply:
I took a look, but could only find pluralizing single words

Reply:
Arghh, and this is the time when using i18n would actually make our lives easier. https://stackoverflow.com/questions/8790093/how-to-pluralize-there-is-are-n-object-objects

Fine, let's keep it the way it is, sad but I don't think it's easily fixable. Unless someone experienced with Rails has a quick solution.